api_app/analyzers_manager/observable_analyzers/stalkphish.py

```diff
@@ -0,0 +1,56 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import requests
+
+from api_app.analyzers_manager import classes
+from api_app.exceptions import AnalyzerRunException
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class Stalkphish(classes.ObservableAnalyzer):
+    base_url: str = "https://api.stalkphish.io/api/v1/"
+
+    def set_params(self, params):
+        self.__api_key = self._secrets["api_key_name"]
+
+    def run(self):
+        headers = {
+            "User-Agent": "Stalkphish/IntelOwl",
+            "Authorization": f"Token {self.__api_key}",
+        }
+        obs_clsfn = self.observable_classification
+
+        if obs_clsfn == self.ObservableTypes.DOMAIN:
+            uri = f"search/url/{self.observable_name}"
+        elif obs_clsfn == self.ObservableTypes.IP:
+            uri = f"search/ipv4/{self.observable_name}"
+        elif obs_clsfn == self.ObservableTypes.URL:
+            uri = f"search/url/{self.observable_name}"
+        elif obs_clsfn == self.ObservableTypes.GENERIC:
```

Comment:
even if it does not work for free user, I would use the endpoint `brand` here because more appropriate for generic strings.

Reply:
I think it is much more relevant and interesting to use the url endpoint because you can search for a string which appear in a URL/URI. The brand endpoint is just reduced to a brand name.

Reply:
makes sense. Can you please aggregate it to the other 2 that leverges the same endpoint?